rfcs/0002-upload-from-dropbox/text.md

```diff
@@ -307,6 +338,142 @@ Checks the status of an existing upload job.
 | 401  | If _dataset_id_ does not exist, the user does not own the collection associated with the dataset, or upload is in-progress. |
 | 400  | If the parameters supplied are invalid.                                                                                     |
 
+##### Status Enumerations
+The statuses in the 200 response are enums defined in the [database](#database-schema):
+
+**upload_status**
+| Values         | Description                                                                                 |
+| -------------- | ------------------------------------------------------------------------------------------- |
+| Waiting        | The upload is enqueued, and waiting for the upload container.                               |
+| Uploading      | The file is actively being uploaded.                                                        |
+| Uploaded       | The upload was completed successfully.                                                      |
+| Failed         | The upload has failed. A new link or file must be uploaded. Any upload progress is deleted. |
+| Cancel Pending | The upload is in the process of being canceled.                                             |
+| Canceled       | The upload has been canceled. Any upload progress is deleted.                               |
+
+**validation_status**
+| Values         | Description                                                                                 |
+| -------------- | ------------------------------------------------------------------------------------------- |
+| Validating     | The validation script is running.                                                           |
+| Valid          | The uploaded file successfully passed validation.                                           |
```

Comment:
@mckinsel Another question, from @hthomas-czi 's design, ideally once validation is complete, we want FE to pull the dataset's metadata and display them in the dataset row. So just wanted to confirm the BE only returns `Valid` when the dataset's metadata has been populated in the database and available for FE 😆 Thank you!

Reply:
We can do that if it helps FE 👍